tika-core/src/main/java/org/apache/tika/transcribe/Transcriber.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Please change name of interface from `Transcriber.java` to `Transcribe.java`
Why? 
The Interface doesn't do the transcribing... the implementation does.

Reply:
I was trying to follow a similar pattern to that of the Translator.
The interface for translating is called "Translator".